roles/ovirt-provider-ovn-driver/README.md

```diff
@@ -0,0 +1,42 @@
+oVirt provider OVN driver
+=============================
+
+This role is used for install and configuration of the oVirt provider OVN driver on oVirt hypervisor.
+
+Requirements
+------------
+
+ * Ansible version 2.3
+
+Role Variables
+--------------
+
+| Name                        | Default value  | Description                                                       |
+|-----------------------------|----------------|-------------------------------------------------------------------|
+| host_deploy_cluster_version | 4.2            | The version of the cluster where the host resides.  oVirt provider OVN driver is installed and configured on versions >= 4.2. |
```

Comment:
I think we could introduce an alias for this variable. I know it's named like this because it's executed from the engine. Maybe you can consider adding such default value:

```yaml
ovn_cluster_version: "{{ host_deploy_cluster_version | default('4.2') }}"
```

It will consider `host_deploy_cluster_version` when passed and if not it will set default value to 4.2. If `ovn_cluster_version` is passed by user it will used. But that's just suggestion for better consistency of variables names, not a must.

Reply:
You can document it then as:

| Name                        | Default value  | Description                                                       |
|-----------------------------|----------------|-------------------------------------------------------------------|
| ovn_cluster_version<br/> alias: <i>host_deploy_cluster_version</i> | 4.2            | The version of the cluster where the host resides.  oVirt provider OVN driver is installed and configured on versions >= 4.2. |